src/constants/trade-config.js

```diff
@@ -1,19 +1,32 @@
-export const getContractTypesConfig = (symbol) => ({
+export const getContractTypesConfig = () => ({
   rise_fall: {
     title: "Rise/Fall",
     trade_types: ["CALL", "PUT"],
+    basis: ["stake", "payout"],
+    components: ["start_date"],
+    barrier_count: 0,
   },
-  high_low: {
-    title: "Higher/Lower",
-    trade_types: ["CALL", "PUT"],
-  },
-  even_odd: {
-    title: "Even/Odd",
-    trade_types: ["DIGITODD", "DIGITEVEN"],
+  rise_fall_equal: {
+    title: "Rise/Fall",
+    trade_types: ["CALLE", "PUTE"],
+    basis: ["stake", "payout"],
+    components: ["start_date"],
+    barrier_count: 0,
+    hide_from_dropdown: true,
   },
-  run_high_low: {
-    title: "Only Ups/Only Downs",
-    trade_types: ["RUNHIGH", "RUNLOW"],
+});
+
+export const getContractCategoriesConfig = () => ({
+  "Ups & Downs": {
```

Comment:
we do maintain a set of supported contract categories in our side, but since we have only rise and fall , just kept the trade types under ups and downs